src/blocked.c

```diff
@@ -591,8 +591,8 @@ void handleClientsBlockedOnKeys(void) {
             updateCachedTime(0);
 
             /* Serve clients blocked on the key. */
-            robj *o = lookupKeyWrite(rl->db,rl->key);
-
+            robj *o = lookupKeyReadWithFlags(rl->db, rl->key, (LOOKUP_NONOTIFY |
+                                                               LOOKUP_NOSTATS));
```

Comment:
i think this looks better.
i also dislike indenting to a specific offset since it'll need to get adjusted if any other argument is added or the function renamed. so if i had to indent it, i would just indent it by 4
```suggestion
            robj *o = lookupKeyReadWithFlags(rl->db, rl->key, (LOOKUP_NONOTIFY | LOOKUP_NOSTATS));
```

Reply:
Sure. I like when lines are not more than 80 so they fit in a half-screen wide editor window on laptop, but sure. Without parentheses even better. (I added them just to make automatic indentation work.)

Reply:
i guess it depends on your laptop 8-).
what about screen split to 3? or a 10" laptop?

anyway, jokes aside we don't have a strict limit, each piece should be formatted so that it (and the code surrounding it) is readable.

in this case i think the line wrap just make is less readable, but i won't insist on it.

Reply:
In this case, a line wrap makes it less readable, except if you get a soft line wrap in a different place because of your old school terminal width. ;-)

I know there's no strict limit, but I was mislead by how it looks for me.

80x24 is a very old standard for terminals, because it is the same size as the punch cards. It's the most common according to [this report from 1975](http://bitsavers.org/pdf/ventureDevelCorp/Alphanumeric_and_Graphic_CRT_Terminals_1975-1980_Nov1975.pdf).

![image](https://user-images.githubusercontent.com/273886/143270895-c1142cd8-a63a-4f9a-9233-85630664bd8b.png)

Reply:
LOL that's old.

maybe the difference between us is that i don't use word wrapping. if a line is too long (in case i'm using a side by side diff on a small computer), i just let it go out of view, and scroll left if i really need to.
the majority of times, when reading the code i need to understand what the code does (which functions it calls), and usually don't care much about the details near the end of the line, so i rather get more lines in my view, and better looking indentation (no word wrapping), than see the entire line.

plus, i usually use very large monitors (40" when docked, and 17" when not)
although i also do much of my work on my "Cosmo Communicator" android phone (6")

Reply:
Yes, turning off line wrapping also works, but it's annoying when you type if it scrolls automatically, it's visually not so nice. In some projects it's just not possible to use a small editor window, but with Redis it works well because most of the code is carefully wrapped at 80.

If you're interested, Nowdays, I use a 15" monitor (never docked) and Emacs with fairly large font size. I rarely read code on the phone. (In a previous project I needed two large monitors for remote desktop environments, test frameworks which opened up 10 small terminals to visualize stuff, large diagrams, etc. But not now.)

Can you merge this one now so we can forget about writable replicas? :-)